Classes/UserFunc/GetPowermailFields.php

```diff
@@ -280,4 +285,12 @@ public function getDefaultFieldTypesForQuery()
         $fieldTypes = $this->getDefaultFieldTypes();
         return ArrayUtility::getQuotedList($fieldTypes);
     }
+
+    protected function getFormUidFromAjaxRequest() {
+        if (isset(GeneralUtility::_GP('ajax')[0])) {
+            preg_match('/data-[0-9]*-' . ConditionContainer::TABLE_NAME . '-([0-9]*)/', GeneralUtility::_GP('ajax')[0], $matches);
+            $conditionContainer = (int)$matches[1];
+            return $this->getFormUidFromConditionContainer($conditionContainer);
+        }
+    }
```

Comment:
Possibly not needed in 9.5+ when https://review.typo3.org/c/Packages/TYPO3.CMS/+/61480 is merged